core-js/src/test/javascript/config/karma.conf.legacy.js

```diff
@@ -112,7 +112,7 @@ module.exports = function(config) {
 
     // Continuous Integration mode
     // if true, it capture browsers, run tests and exit
-    singleRun: true,
+    singleRun: false,
```

Comment:
Yes, so we can edit/debug the specs and have the browser automatically reload them. This is for the non-CI configuration file only.